### PR TITLE
Code Style MainWindow.vala: More verbose variable names & Initialize Objects with Properties 

### PR DESCRIPTION
**Pull request by andreasfelix**
The first commit replace single character variable names with more verbose name. I used the vala-language server for rename to make sure nothing gets messed up.

The second commit initializes object with properties. I used the script from https://github.com/elementary/gala/pull/965

### CODE REVIEW

File: src/MainWindow.vala [hunk 1]
```diff
@@ -348,75 +354,79 @@ namespace Terminal {
...
-            var zoom_out_button = new Gtk.Button.from_icon_name ("zoom-out-symbolic", Gtk.IconSize.MENU);
-            zoom_out_button.action_name = ACTION_PREFIX + ACTION_ZOOM_OUT_FONT;
-            zoom_out_button.tooltip_markup = Granite.markup_accel_tooltip (
-                application.get_accels_for_action (zoom_out_button.action_name),
-                _("Zoom out")
-            );
+            var zoom_out_button = new Gtk.Button.from_icon_name ("zoom-out-symbolic", Gtk.IconSize.MENU) {
+                action_name = ACTION_PREFIX + ACTION_ZOOM_OUT_FONT,
+                tooltip_markup = Granite.markup_accel_tooltip (
+                application.get_accels_for_action (ACTION_PREFIX + ACTION_ZOOM_OUT_FONT), _("Zoom out"))
```
Comment (danrabbit) [inline src/MainWindow.vala:367]: Indentation seems incorrect here

File: src/MainWindow.vala [hunk 2]
```diff
@@ -348,75 +354,79 @@ namespace Terminal {
...
-            zoom_in_button.action_name = ACTION_PREFIX + ACTION_ZOOM_IN_FONT;
-            zoom_in_button.tooltip_markup = Granite.markup_accel_tooltip (
-                application.get_accels_for_action (zoom_in_button.action_name),
+            var zoom_in_button = new Gtk.Button.from_icon_name ("zoom-in-symbolic", Gtk.IconSize.MENU) {
+                action_name = ACTION_PREFIX + ACTION_ZOOM_IN_FONT,
+                tooltip_markup = Granite.markup_accel_tooltip (
+                application.get_accels_for_action (ACTION_PREFIX + ACTION_ZOOM_IN_FONT),
                 _("Zoom in")
-            );
+            )
```
Comment (danrabbit) [inline src/MainWindow.vala:383]: Indentation seems odd here as well

**Review by danrabbit — CHANGES_REQUESTED** [review]
Hey thank you for doing this code style cleanup! Mostly looks good, just a couple places where indentation seems off

File: src/MainWindow.vala (hunk 2 above)
Comment (andreasfelix) [reply src/MainWindow.vala]: fixed

**Review by danrabbit — APPROVED** [review]
Looks good, nice work!

**Pull request merged**